activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -329,13 +329,13 @@ def initialize(spec)
         # default max pool size to 5
         @size = (spec.config[:pool] && spec.config[:pool].to_i) || 5
 
-        # The cache of threads mapped to reserved connections, the sole purpose
-        # of the cache is to speed-up +connection+ method, it is not the authoritative
-        # registry of which thread owns which connection, that is tracked by
+        # The cache of threads mapped to reserved connections. The sole purpose
```

Comment:
`The cache of threads mapped to reserved connections.` 

This sentence feels incomplete.